lix/packages/client/src/repoState.ts

```diff
@@ -103,41 +230,68 @@ export async function repoState(
 			throw new Error("fs provider does not support placeholders")
 		}
 		console.info("Using lix for cloning repo")
-
+		// FIXME: symlink = false in git/config!???
```

Comment:
can you describe better what fix is needed

Reply:
i just discovered that isogit writes into .git/config    symlink = false.   i have no idea what the impact is or why but as we obviously use symlinks and symlinks work so far it might have implications. but what implications?

Reply:
@martin-lysk  this is another issue that was allready present and just was discovered in this pr, thus is tracked in separate ticket: https://linear.app/opral/issue/LIX-70/symlink-=-false-in-gitconfig